tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -49,10 +49,10 @@ def test_torch_small(self):
     @require_torch
     def test_torch_small_no_tokenizer_files(self):
         # test that model without tokenizer file cannot be loaded
-        with pytest.raises(ValueError):
+        with pytest.raises(OSError):
             pipeline(
                 task="automatic-speech-recognition",
-                model="hf-internal-testing/tiny-random-wav2vec2",
```

Comment:
a tokenizer was recently added to this repo. Also it should give a OSError error IMO